[CI] MultiNodeBackwardsIT.test {p0=cat.tasks/10_basic/Test cat tasks output} fails

See: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+multijob-os-compatibility/os=centos/243/console

Looks like it may be a simple regex issue:

```
Suite: org.elasticsearch.backwards.MultiNodeBackwardsIT
  1> [2016-04-13 08:46:41,455][ERROR][org.elasticsearch.test.rest.client] Adding header Accept
  1>  with value application/yaml
  1> [2016-04-13 08:47:09,457][ERROR][org.elasticsearch.test.rest.client] Adding header Accept
  1>  with value application/yaml
  2> REPRODUCE WITH: gradle :qa:backwards-5.0:integTest -Dtests.seed=2C193B1BE603DD5E -Dtests.class=org.elasticsearch.backwards.MultiNodeBackwardsIT -Dtests.method="test {p0=cat.tasks/10_basic/Test cat tasks output}" -Des.logger.level=WARN -Dtests.security.manager=true -Dtests.locale=is-IS -Dtests.timezone=Mexico/General
FAILURE 0.02s | MultiNodeBackwardsIT.test {p0=cat.tasks/10_basic/Test cat tasks output} <<< FAILURES!
   > Throwable #1: java.lang.AssertionError: field [$body] was expected to match the provided regex but didn't
   > Expected:  # action  task_id      parent_task_id      type    start_time   timestamp            running_time  ip                                  node
   > ^(  \S+\s+  \S+\:\d+\s+  (?:\-|\S+\:\d+)\s+  \S+\s+  \d+\s+       \d\d\:\d\d\:\d\d\s+  \S+\s+        \d{1,3}.\d{1,3}.\d{1,3}.\d{1,3}\s+  \S+(?:\s\S+)*\n)+$
   >      but: was "cluster:monitor/tasks/lists    sEat2lQHTJ26PwkjuXqDXw:9695 -                           transport 1460537251333 08:47:31 83.2micros 127.0.0.1 Grasshopper \ncluster:monitor/tasks/lists[n] sEat2lQHTJ26PwkjuXqDXw:9696 sEat2lQHTJ26PwkjuXqDXw:9695 direct    1460537251333 08:47:31 21.7micros 127.0.0.1 Grasshopper \ncluster:monitor/tasks/lists[n] S8fYxQvLQwuEhXfQ2S5szw:4794 sEat2lQHTJ26PwkjuXqDXw:9695 netty     1460537251333 08:47:31 27.5micros 127.0.0.1 Mad-Dog\n"
   > 	at __randomizedtesting.SeedInfo.seed([2C193B1BE603DD5E:A44D04C148FFB0A6]:0)
```

--- a/core/src/main/resources/config/names.txt
+++ b/core/src/main/resources/config/names.txt
@@ -1047,7 +1047,7 @@ Gosamyr
 Grand Director
 Grandmaster
 Grappler
-Grasshopper 
+Grasshopper
 Grasshopper II
 Graviton
 Gravity
@@ -1483,7 +1483,7 @@ Lightmaster
 Lightspeed
 Lila Cheney
 Lilandra Neramani
-Lilith, the Daughter of Dracula 
+Lilith, the Daughter of Dracula
 Lin Sun
 Link
 Lionheart

